packages/platform-ios/native_modules.rb

```diff
@@ -55,10 +57,10 @@ def use_native_modules!(root = "..", packages = nil)
     end
 
     # Use relative path
-    absolute_podspec_path = File.dirname(podspec_path)
-    relative_podspec_path = File.join(root, absolute_podspec_path.partition('node_modules').last(2).join())
+    folder = File.dirname(podspec_path)
+    path = folder.partition(project_root).last()
```

Comment:
```suggestion
    path = folder.split(project_root).last
```

I _believe_ that `partition` is an activesupport extension, whereas `split` is core.

Reply:
right, even if `partition` is core I think `split` is more descriptive and straight-forward

Reply:
`split().last` returns `nil` when the left and right side is the same, while `partition.last()` converts this to empty string. 

I pushed a fix for that in 500a868